homeassistant/components/group/strings.json

```diff
@@ -283,20 +283,20 @@
   },
   "issues": {
     "uoms_not_matching_device_class": {
-      "title": "Unit of measurements are not correct",
-      "description": "Unit of measurements `{uoms}` of input sensors `{source_entities}` are not compatible and can't be converted with the device class `{device_class}` of sensor group `{entity_id}`.\n\nPlease correct the unit of measurements on the source entities and reload the group sensor to fix this issue."
+      "title": "Units of measurement are not correct",
+      "description": "Units of measurement `{uoms}` of input sensors `{source_entities}` are not compatible and can't be converted with the device class `{device_class}` of sensor group `{entity_id}`.\n\nPlease correct the unit of measurement on the source entities and reload the group sensor to fix this issue."
     },
     "uoms_not_matching_no_device_class": {
-      "title": "Unit of measurements is not correct",
-      "description": "Unit of measurements `{uoms}` of input sensors `{source_entities}` are not compatible when not using a device class on sensor group `{entity_id}`.\n\nPlease correct the unit of measurements on the source entities or set a proper device class on the sensor group and reload the group sensor to fix this issue."
+      "title": "Units of measurement are not correct",
+      "description": "Units of measurement `{uoms}` of input sensors `{source_entities}` are not compatible when not using a device class on sensor group `{entity_id}`.\n\nPlease correct the unit of measurement on the source entities or set a proper device class on the sensor group and reload the group sensor to fix this issue."
     },
     "device_classes_not_matching": {
-      "title": "Device classes is not correct",
-      "description": "Device classes `{device_classes}` on source entities `{source_entities}` needs to be same for sensor group `{entity_id}`.\n\nPlease correct the device classes on the source entities and reload the group sensor to fix this issue."
+      "title": "Device class is not correct",
+      "description": "Device class `{device_classes}` on source entities `{source_entities}` needs to be same for sensor group `{entity_id}`.\n\nPlease correct the device classes on the source entities and reload the group sensor to fix this issue."
     },
     "state_classes_not_matching": {
-      "title": "State classes is not correct",
-      "description": "State classes `{state_classes}` on source entities `{source_entities}` needs to be same for sensor group `{entity_id}`.\n\nPlease correct the state classes on the source entities and reload the group sensor to fix this issue."
+      "title": "State class is not correct",
```

Comment:
I believe `device_classes` and `state_classes` are a list of values seperated by comma's, so the sentence would be
> Device class `measurement, total, total_increasing` on ...

Does that sound logical?

Reply:
Right, if they differ this will be a list, obviously.
Then the "needs" is wrong and we make everything plural:

"XYZ classes … need to be identical" instead of "… needs to be same" (which is bad grammar anyhow)